### PR TITLE
Fixes the panic in the wrapping layer

### PR DESCRIPTION
**Pull request by andresuribe87**
Be advised, I have no idea what I'm doing with the xcode project related stuff.

### CODE REVIEW

File: identity/did.go [hunk 1]
```diff
@@ -21,9 +23,17 @@ type DIDKeyWrapper struct {
 // if !ok { ... }
 func GenerateDIDKey(kt string) (*DIDKeyWrapper, error) {
 	privateKey, didKey, err := did.GenerateDIDKey(stringToKeyType(kt))
+	jwkKey, err := jwk.New(privateKey)
```
Comment (decentralgabe) [inline identity/did.go:26]: clever

**Review by decentralgabe — APPROVED** [review]
see note about secp

File: example/ios/example.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -51,7 +51,7 @@
 		84EC056A29935E41001904E4 /* exampleUITests.xctest */ = {isa = PBXFileReference; explicitFileType = wrapper.cfbundle; includeInIndex = 0; path = exampleUITests.xctest; sourceTree = BUILT_PRODUCTS_DIR; };
 		84EC056E29935E41001904E4 /* exampleUITests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = exampleUITests.swift; sourceTree = "<group>"; };
 		84EC057029935E41001904E4 /* exampleUITestsLaunchTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = exampleUITestsLaunchTests.swift; sourceTree = "<group>"; };
-		84EC058429936A3A001904E4 /* Identity.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = Identity.xcframework; path = ../../../Identity.xcframework; sourceTree = "<group>"; };
+		84EC058429936A3A001904E4 /* Identity.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = Identity.xcframework; path = ../../Identity.xcframework; sourceTree = SOURCE_ROOT; };
```
Comment (shamilovtim) [inline example/ios/example.xcodeproj/project.pbxproj:54]: not a huge deal but i'd probably just hard reset this file

File: example/ios/example.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -138,7 +138,8 @@
 			children = (
 				84EC058429936A3A001904E4 /* Identity.xcframework */,
 			);
-			path = Frameworks;
+			name = Frameworks;
+			path = ../../Identity.xcframework;
```
Comment (shamilovtim) [inline example/ios/example.xcodeproj/project.pbxproj:142]: When you went to build in Xcode were you having an issue with the Framework not showing up?

Comment (andresuribe87) [reply example/ios/example.xcodeproj/project.pbxproj:142]: Yes :/

Comment (shamilovtim) [reply example/ios/example.xcodeproj/project.pbxproj:142]: Weird. Thanks for fixing.

**Review by shamilovtim — APPROVED** [review]

**Pull request merged**